contracts/core/Core.sol

```diff
@@ -668,18 +644,6 @@ contract Core is MasterCopyNonUpgradable, ConsensusModule, MosaicVersion, CoreSt
     {
         assert(precommit != bytes32(0));
```

Comment:
Maybe we should pass `_precommit` in the params and check if its equal to the `precommit`.

Reply:
Why do you think we should do it (it's not going to harm if we do, except paying a gas). The thinking here is that `Consensus` does all checks and as this function is guarded by `onlyConsensus` we do not need to do any check on that regard. What are your thoughts?

Reply:
I think it is ok. We trust consensus. 👍